Comp/r-sas_correlation.qmd

```diff
@@ -0,0 +1,49 @@
+## R vs SAS Correlation
+
+## Codes
```

Comment:
Would it be helpful to describe the data that you used? 

I think it's the same as your SAS contribution, right?

Reply:
Done Orla!

Reply:
Looks good to me - thanks! 
cc @statasaurus